unstructured_ingest/v2/unstructured_api.py

```diff
@@ -87,13 +87,16 @@ async def call_api_async(
     """
     from unstructured_client import UnstructuredClient
 
+    # Note(austin) - the sdk takes the base url, but users may pass the full endpoint
+    # For consistency, strip off the path when it's given
+    base_url = server_url[:-19] if "/general/v0/general" in server_url else server_url
```

Comment:
Would be safer to split on `/general/v0/general` in case there are trailing spaces or tabs.

Reply:
Also `server_url` is an optional parameter, so it would make sense to check if it exists.